pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -137,15 +171,6 @@ func (c *KubeAPIServerServerPatchContext) PatchServer(server *master.Master) err
 		}
 		return nil
 	})
-	server.GenericAPIServer.AddPostStartHookOrDie("openshift.io-restmapperupdater", func(context genericapiserver.PostStartHookContext) error {
```

Comment:
where is this gone? Why don't we need that anymore?

Reply:
> where is this gone? Why don't we need that anymore?

we use the upstream restmapper now.